unitOneAssessmentPractice.js

```diff
@@ -1,77 +1,181 @@
 let assert = require('assert')
```

Comment:
Great work with this review. It clearly demonstrates that you have good understanding of JS logics and good grasp on different methods.